ideascube/mediacenter/models.py

```diff
@@ -60,7 +60,7 @@ class Document(SearchMixin, TimeStampedModel):
                                 upload_to='mediacenter/preview',
                                 max_length=10240,
                                 blank=True)
-    credits = models.CharField(verbose_name=_('credit'), max_length=300)
+    credits = models.CharField(verbose_name=_('autorship'), max_length=300)
```

Comment:
I'm wondering about the all lower-casing here... In other `verbose_name`s we sometimes capitalize the first letter.

Maybe we should make a pass on all these at once eventually.